app/renderer/components/common/contextMenu/contextMenu.js

```diff
@@ -215,7 +218,7 @@ class ContextMenu extends React.Component {
     const currentWindow = state.get('currentWindow')
     const activeFrame = frameStateUtil.getActiveFrame(currentWindow) || Immutable.Map()
     const selectedIndex = currentWindow.getIn(['ui', 'contextMenu', 'selectedIndex'], null)
-    const contextMenuDetail = currentWindow.get('contextMenuDetail', Immutable.Map())
+    const contextMenuDetail = contextMenuState.getContextMenu()
```

Comment:
we should still pass in the state even though we aren't using it. That will simplify things if we fix the real issue later.

Reply:
the idea is to hide the hack from everything except the action dispatch